tests/test_column.py

```diff
@@ -359,6 +359,35 @@ def test_segmented(num_files, num_segments_per_file, values):
     pd.testing.assert_series_equal(table.df[column_id], series)
 
 
+@pytest.mark.parametrize(
+    'timezone, values, expected_dates',
+    [
+        (
+            'UTC',
+            [1],
+            [pd.Timestamp('1970-01-01T00:00:00.000000001')],
+        ),
+        (
+            'Europe/Berlin',
```

Comment:
This seems to always use winter time. As at the moment we are 2 hours in front of UTC, but `pandas` only returns 1 hour difference.

Reply:
But doesn't Januar 1970 fall into winter time?